pages/api/startHostedActivitySessionViaHostedPagesLink/[hostedPagesLinkId].ts

```diff
@@ -53,10 +56,10 @@ export default async function handler(
 
     const session_response = await session.json()
 
-    const { session_id } =
+    const { session_id, language } =
```

Comment:
[Q] @mohsinht Why aren't we just using `session_url` and let the back-end take care of returning the correct URL (including the correct language prefix)? If I am not mistaken, this is also how it works for `startHostedPathwaySession` and `startHostedActivitySession`?

That way we don't need a `mapLanguageToLocalePrefix` in the front-end either.

## API call

```graphql
mutation StartHostedActivitySessionViaHostedPagesLink($input: StartHostedActivitySessionViaHostedPagesLinkInput!) {
  startHostedActivitySessionViaHostedPagesLink(input: $input) {
    session_url
  }
}
```

## StartHostedActivitySessionFlow.tsx

```javascript
window.location.href = session_url
```